src/app/client/package.json

```diff
@@ -77,7 +77,7 @@
     "@project-sunbird/sunbird-quml-player-v9": "5.1.5",
     "@project-sunbird/sunbird-resource-library": "7.0.3",
     "@project-sunbird/sunbird-video-player-v9": "5.5.1",
-    "@project-sunbird/telemetry-sdk": "0.0.29",
+    "@project-sunbird/telemetry-sdk": "1.3.0",
```

Comment:
@sedin-tushar did u test the telemetry related functionality post this changes

Reply:
How can  i specifically check telemetry-functionalities @princegupta1131

Reply:
The telemetry-sdk library is downgraded to previous version itself